assets/js/blocks/featured-category/block.js

```diff
@@ -70,6 +77,14 @@ const FeaturedCategory = ( {
 	debouncedSpeak,
 	triggerUrlUpdate = () => void null,
 } ) => {
+	const { gradientValue, setGradient } = useGradient();
+	const onResize = useCallback(
```

Comment:
Do you think it is worth debouncing this callback if it's going to be executed on every `resize` event?

Reply:
Actually it looks like it's already taken care of within the `<ConstrainedResizable />` component, but if you could just confirm?

Reply:
Yep, I can confirm. I've added `useThrottle` in [this commit](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/6181/commits/73b3846bc28e2d1471555ceff98a1a6832932068) to make it easier to create throttled callbacks. I decided to go with throttling instead of debouncing here, because the approach in this case felt a bit more appropriate and responsive.

Gutenberg doesn't do this, by the way, and I was thinking whether I should make a PR to things like the Cover block to throttle/debounce their resize events at some point.